command/sub8_launch/launch/subsystems/blueview.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+    <node pkg="blueview_driver" type="blueview_driver" name="blueview_driver" >
+        <rosparam>
+            frame_id: blueview
+            device: 192.168.37.53
```

Comment:
Don't know the domain and not sure if the sdk handles domains, but I'm happy to try it.